bundles/org.openhab.binding.velbus/README.md

```diff
@@ -36,15 +36,27 @@ Once the bridge has been added as a thing, a manual scan can be launched to disc
 
 ## Thing Configuration
 
-The Velbus bridge needs to be added first in the things file or through Paper UI.
+The Velbus Serial bridge needs to be added first in the things file or through Paper UI.
 It is necessary to specify the serial port device used for communication.
 On Linux systems, this will usually be either `/dev/ttyS0`, `/dev/ttyUSB0` or `/dev/ttyACM0` (or a higher  number than `0` if multiple devices are present).
 On Windows it will be `COM1`, `COM2`, etc.
 
-In the things file, this looks e.g. like
+In the things file, this might look like
 
 ```
-Bridge velbus:bridge:1 [ port="COM1" ]
+Bridge velbus:bridge:VelbusSerialBridge [ port="COM1" ]
+```
+
```

Comment:
Since the documentation is versioned, this can be removed if the bridge is available in the current version.